README.md

```diff
@@ -1,11 +1,29 @@
 # cheri-examples
 cheri-riscv sample c programs
 
+------
 ## Build instructions with CMake
+Build al the example code with CMake using  default settings :
```

Comment:
s/al/all/ ? Also notice the double space before "default".

Reply:
Guilty! Fixed. will push them soon.